### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by sharadregoti**

### CODE REVIEW

File: modules/auth/schema/inspection_test.go [hunk 1]
```diff
@@ -75,7 +75,11 @@ func TestSchema_schemaInspection(t *testing.T) {
 				project:   tt.fields.project,
 			}
 
+<<<<<<< HEAD
```
Comment (YourTechBud) [inline modules/auth/schema/inspection_test.go]: Merge conflict remains

File: modules/crud/sql/describe.go [hunk 2]
```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/spaceuptech/space-cloud/utils"
 )
 
-// not to be exposed externally
+// NOTE: not to be exposed externally
```
Comment (YourTechBud) [inline modules/crud/sql/describe.go]: Comments are wrong. Always start with the function name you are describing. Everything else goes below that

File: utils/handlers/graphql.go [hunk 3]
```diff
@@ -50,3 +51,28 @@ func HandleGraphQLRequest(graphql *graphql.Module) http.HandlerFunc {
...
+		// Create a context of execution
+		ctx, cancel := context.WithTimeout(r.Context(), 5*time.Second)
+		defer cancel()
+
+		vars := mux.Vars(r)
+		dbType := vars["dbType"]
+		col := vars["col"]
+		project := vars["project"]
+
+		result, err := schema.SchemaInspection(ctx, dbType, project, col)
```
Comment (YourTechBud) [inline utils/handlers/graphql.go]: You should do some authentication before this step. Look at read config handler in `handlers/config.go`. we just want to check for a valid JWT token

File: utils/server/routes.go [hunk 4]
```diff
@@ -31,6 +31,9 @@ func (s *Server) routes(router *mux.Router, profiler bool, staticPath string) {
 	router.Methods("POST").Path("/v1/api/config/static").HandlerFunc(handlers.HandleStoreStaticConfig(s.adminMan, s.syncMan))
 	router.Methods("DELETE").Path("/v1/api/config/{project}").HandlerFunc(handlers.HandleDeleteProjectConfig(s.adminMan, s.syncMan, s.configFilePath))
 
+	//Initialize route for graphql schema inspection
+	router.Methods("GET").Path("/v1/api/config/inspect/{dbType}/{project}/{col}").HandlerFunc(handlers.HandleInspectionRequest(s.schema))
```
Comment (YourTechBud) [inline utils/server/routes.go]: Project should always come before dbtype

**Comment by sharadregoti** [comment]
unexported all the exported variables in schema package

**Pull request merged**